README.md

```diff
@@ -140,6 +167,8 @@ whenever possible: the only time we will dispatch a job to a worker thread when
 already running is when a new command arrives, the category reserves M threads, and the thread pool
 is currently processing fewer than M jobs for that category.
 
+[//]: # Could just dedicate a thread for message handling?
```

Comment:
There is always such a thread, the "proxy" thread.  All the threads configured here are for actually handling the responses, and it's the proxy thread that does that allocation based on the configured parameters.